src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy

```diff
@@ -388,7 +387,6 @@ class DevTask extends AbstractFeatureTask {
                     null /* compileOptions not needed since useBuildRecompile is true */, keepTempContainerfile, mavenCacheLocation, projectModuleList /* multi module upstream projects */,
                     false /* recompileDependencies only supported in ci.maven */, packagingType, buildFile, null /* parent build files */, generateFeatures, null /* compileArtifactPaths */, null /* testArtifactPaths */, webResourceDirs /* webResources */
                 );
-            this.upstreamGradleProjects = gradleProjects;
```

Comment:
I don't understand why you made all these changes unrelated to windows and the yaml. Only the test case needed to be fixed for Windows and Java 8.

Reply:
Removed the allProjects from this PR. Will add this later when its being used